deployments/aad-azure-Identity-binding.yml

```diff
@@ -0,0 +1,8 @@
+apiVersion: "aadpodidentity.k8s.io/v1"
```

Comment:
can we use `.yaml` extension everywhere instead of `.yml` please

Reply:
done